rules/repositories.bzl

```diff
@@ -93,10 +93,24 @@ native_binary(
     out = "index-import",
     visibility = ["//visibility:public"],
 )
+
+native_binary(
+    name = "validate_index",
```

Comment:
want to learn what's this from (new bazel versions' requirement?)

Reply:
No, not required, but will make debugging using these tools easier since now we can just `bazel run` them (they were added with the bump to index-import)